app/cli/internal/action/referrer_discover.go

```diff
@@ -35,13 +38,29 @@ type ReferrerItem struct {
 	References   []*ReferrerItem `json:"references"`
 }
 
-func NewReferrerDiscover(cfg *ActionsOpts) *ReferrerDiscover {
+func NewReferrerDiscoverPrivate(cfg *ActionsOpts) *ReferrerDiscover {
 	return &ReferrerDiscover{cfg}
```

Comment:
Maybe here we have some nomenclature inconsistency?
xyzPrivate vs. xyzPublic everywhere, except here: the public has the "Public" suffix, the private it simply doesn't have suffix